packages/cli/commands/config/set.js

```diff
@@ -1,25 +1,93 @@
-const { addConfigOptions, addAccountOptions } = require('../../lib/commonOpts');
-const defaultAccount = require('./set/defaultAccount');
-const defaultMode = require('./set/defaultMode');
-const httpTimeout = require('./set/httpTimeout');
-const allowUsageTracking = require('./set/allowUsageTracking');
+const { loadAndValidateOptions } = require('../../lib/validation');
 const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { getAccountId } = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+const { setDefaultMode } = require('./set/defaultMode');
+const { setHttpTimeout } = require('./set/httpTimeout');
+const { setAllowUsageTracking } = require('./set/allowUsageTracking');
 
 const i18nKey = 'cli.commands.config.subcommands.set';
 
 exports.command = 'set';
 exports.describe = i18n(`${i18nKey}.describe`);
 
-exports.builder = yargs => {
-  addConfigOptions(yargs, true);
-  addAccountOptions(yargs, true);
+const selectOptions = async () => {
+  const { mode } = await promptUser([
+    {
+      type: 'list',
+      look: false,
+      name: 'mode',
+      pageSize: 20,
+      message: i18n(`${i18nKey}.promptMessage`),
+      choices: [
+        { name: 'Default mode', value: { defaultMode: '' } },
+        { name: 'Allow usage tracking', value: { allowUsageTracking: '' } },
+        { name: 'HTTP timeout', value: { httpTimeout: '' } },
+      ],
+    },
+  ]);
+
+  return mode;
+};
+
+const handleConfigUpdate = async (accountId, options) => {
```

Comment:
This is the only function that I'm questioning a bit; it seems a bit off to be checking that the options aren't `undefined`, but I also can't come up with any way to refactor 🤷‍♀️ Just thought I would leave a quick note.

Reply:
Yeah I have to do this because some of them are booleans. So I can't just check for `!<option>`